src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    public static Ball getRandomBall() {
+        String color = ColorSupplier.getRandomColor();
+        Random random = new Random();
+        int number = random.nextInt(100);
```

Comment:
100 should be made a constant field (private static final field with name in UPPER_CASE)
https://www.javatpoint.com/java-constant